src/main/php/xp/runtime/Evaluate.class.php

```diff
@@ -26,6 +27,16 @@ public static function main(array $args) {
       $code= new Code($args[0]);
     }
 
-    return $code->run($code->fragment(), [XPClass::nameOf(self::class)] + $args);
+    try {
+      return $code->run($code->fragment(), [XPClass::nameOf(self::class)] + $args);
+    } catch (ModuleNotFound $e) {
+      Console::$err->writeLine("\033[41;1;37mError: ", $e->getMessage(), "\033[0m");
+      Console::$err->writeLinef(
+        "Try installing it via `\033[36mmkdir -p '%1\$s' && composer require -d '%1\$s' %2\$s\033[0m`",
+        Environment::configDir('xp').$code->namespace(),
```

Comment:
If the namespace contains `\`, this should be properly translated to `/` on Un\*x systems